### PR TITLE
fsdiffer: imported as subtree

### PR DESCRIPTION
**Pull request by dgonyeo**
Imported fsdiffer from https://github.com/dgonyeo/fsdiffer as a subtree
into util/fsdiffer

### CODE REVIEW

**Comment by jonboulle** [comment]
@sgotti shall we assign you copyright here? it'd be dual as we've now patched it:
```
// Copyright Simone Gotti 2015
// Copyright CoreOS, Inc 2015
```

File: util/fsdiffer/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[._]*.s[a-w][a-z]
```
Comment (jonboulle) [inline util/fsdiffer/.gitignore]: I think you can omit this now

**Comment by dgonyeo** [comment]
For what it's worth, as far as dual copyright goes I didn't actually modify any of his files. Just added a couple.

**Comment by jonboulle** [comment]
Couple of comments. Also, can you add copyright headers?  LGTM after that.

**Pull request merged**